mapswipe_workers/tests/unittests/fixtures/change_detection/project_drafts.json

```diff
@@ -0,0 +1,111 @@
+{
+  "change_detection_default_with_bing": {
+        "createdBy": "Sample Manager",
+        "geometry": {"type":"FeatureCollection","features":[{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-175.21785736083984,-21.122295110595505],[-175.2367401123047,-21.148873980682744],[-175.21339416503906,-21.152716314425852],[-175.19931793212888,-21.15239612375494],[-175.19588470458984,-21.147913381670975],[-175.1931381225586,-21.136385707660683],[-175.1934814453125,-21.129660817021904],[-175.21785736083984,-21.122295110595505]]]}}]},
+        "image": "http://www.redcrosseth.org/media/k2/items/cache/5a05a447acfdf6fcc40548cc4c1cea8d_L.jpg",
+        "lookFor": "DESTROYED BUILDINGS",
+        "name": "Change Detection Sample Project",
+        "projectDetails": "This project uses Bing as the tile server and zoom level 18 for the before image. For after we use imagery from open aerial map.",
+        "verificationNumber": 3,
+        "groupSize": 15,
+        "projectType": 3,
+        "tileServerA": {
```

Comment:
we can clean up this part a bit and remove unused attributes such as `caption `apiKeyRequired`, `date`